krita_batch_exporter/Infrastructure.py

```diff
@@ -14,8 +14,33 @@
 
 
 def dataToPIL(wnode):
-    img = wnode.node.projectionPixelData(*wnode.bounds).data()
-    img = Image.frombytes("RGBA", wnode.size, img, "raw", "BGRA", 0, 1)
+    [x, y, w, h] = wnode.bounds
+    # Create a temporary Document for color space conversion. This document will have
+    # one transparent layer,
```

Comment:
Could you use a docstring at the top of the function instead of inline comments? That's the convention in Python, and that way it will show up in auto-completion.

E.g.
> Returns an Image object using the Pillow library with 8-bit sRGB data that Pillow can then process.
> If the input node's color space isn't 8-bit sRGB, uses Krita to convert the document to the right color space.

Reply:
Added the suggested docstring. Thanks for the help!